app/controllers/inventories_controller.rb

```diff
@@ -2,10 +2,11 @@
 
 class InventoriesController < ApplicationController
   before_action :set_inventory, only: %i[destroy]
+  before_action :set_user
```

Comment:
Remember to specify which methods to include or exclude to avoid unnecessary queries 👀 !